src/PerformanceTests/Tests/Tools/TestEnvironment.cs

```diff
@@ -153,7 +153,8 @@ static void Clone(string src, string dst)
                 if (equalTimestamps) return;
                 File.Delete(dst);
             }
-            if (!SymbolicLink.Create(src, dst)) File.Copy(src, dst);
+
+            File.Copy(src, dst);
```

Comment:
@danielmarbach You probably meant this `File.Copy`, yes, here it probably is a good idea to allow overwrites.